src/remote-control/process-update-delegate.js

```diff
@@ -146,7 +206,32 @@ export default class ProcessUpdateDelegate {
             const from = presence.getAttribute('from');
 
             if (from === this.getSpotId()) {
+                // When Spot has left, trigger screenshare cleanup to stop any
+                // active {@code ScreenshareConnection} and its associated
+                // media.
+                this.stopScreenshare();
+
                 this._store.dispatch(setSpotLeft());
+            } else {
+
+                // When a remote leaves, notify the Jitsi-Meet meeting so that
+                // it can trigger any cleanup of active direct connections to a
+                // remote control.
+                const iq = $iq({ type: 'set' })
+                    .c('jingle', {
+                        xmlns: 'urn:xmpp:jingle:1',
+                        action: 'unavailable'
+                    })
+                    .c('details')
+                    .t('unavailable')
+                    .up();
+                const stringifiedIq
+                    = new XMLSerializer().serializeToString(iq.nodeTree);
```

Comment:
is this what we use in the lib?  i don't remember seeing it...i assumed the strophe types had a member method to serialize

Reply:
strophe does have toString on its builder I should use.